doctr/models/factory/hub.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2022, Mindee.
```

Comment:
Here, it's not starting by 2021. Should the action fail or should it only verify that the year is greater than `starting_year` defined in action ?

Reply:
second case it need to be bigger so two ways are valid 2021-x (if file was created earlier) or 202x

Reply:
Ok thanks for clarification !